### PR TITLE
Fake marks final

### PR DESCRIPTION
**Pull request by david-yz-liu**

### CODE REVIEW

File: lib/tasks/marks.rake [hunk 1]
```diff
@@ -0,0 +1,25 @@
+namespace :db do
+
+  desc "Create fake marks for assignments"
+  task :marks => :environment do
+    puts "Assign Marks for Assignments"
+    Grouping.all.each do |grouping|
+      time = grouping.assignment.submission_rule.calculate_collection_time.localtime
+        new_submission = Submission.create_by_timestamp(grouping, time)
+        result = new_submission.results.first
+        mark = Mark.create(result_id: result.id,
```
Comment (david-yz-liu) [inline lib/tasks/marks.rake]: Put `result_id: ...` on a new line

File: lib/tasks/marks.rake [hunk 2]
```diff
@@ -0,0 +1,25 @@
+namespace :db do
+
+  desc "Create fake marks for assignments"
```
Comment (david-yz-liu) [inline lib/tasks/marks.rake]: Use single-quotes

File: lib/tasks/marks.rake [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      time = grouping.assignment.submission_rule.calculate_collection_time.localtime
+        new_submission = Submission.create_by_timestamp(grouping, time)
+        result = new_submission.results.first
+        mark = Mark.create(result_id: result.id, 
+          mark: rand(1..5),
+          markable_id: rand(1..5),
+          markable_type: "RubricCriterion",
+          markable: grouping.assignment.rubric_criteria.first)
+        result.marks.push(mark)
+        mark.save
```
Comment (david-yz-liu) [inline lib/tasks/marks.rake]: You shouldn't need this line

File: lib/tasks/marks.rake [hunk 4]
```diff
@@ -0,0 +1,25 @@
+namespace :db do
+
+  desc "Create fake marks for assignments"
+  task :marks => :environment do
+    puts "Assign Marks for Assignments"
+    Grouping.all.each do |grouping|
+      time = grouping.assignment.submission_rule.calculate_collection_time.localtime
+        new_submission = Submission.create_by_timestamp(grouping, time)
```
Comment (david-yz-liu) [inline lib/tasks/marks.rake]: Everything here and down is too indented

**Comment by PatGuan** [comment]
Hey david, this pull req can be closed if you like as I made a new one to avoid conflicts.

**Pull request closed without merging**